SingularityUI/app/components/newDeployForm/NewDeployForm.jsx

```diff
@@ -377,6 +385,14 @@ class NewDeployForm extends Component {
             DOCKER_PORT_MAPPING_FIELDS,
             (id) => portMapping[id] || INDEXED_DOCKER_PORT_MAPPING_FIELDS[id].default));
           deployObject[fieldId.id] = portMappings;
+        } else if (fieldId.type === 'dockerParameters') {
+          console.log(fieldId)
+          debugger
```

Comment:
What's this line do? debugger? 
Ignore if this is obvious to JS programmers. New language for me.